test-common/src/main/java/com/mentoring/exceptions/ExtraFreeSlotsNumberNotFoundException.java

```diff
@@ -0,0 +1,8 @@
+package com.mentoring.exceptions;
+
+public class ExtraFreeSlotsNumberNotFoundException extends Exception {
```

Comment:
I would be more generic with this one and throw ExpectedAttributeNotFoundException. But thinking further, I'm not sure if exception throwing is the correct way here, I would rather throw a simple AssertionError where this occurs  (so no need for separate class). I believe exception throwing is more appropriate for framework related errors and assertions for application related failures (something failed to meet the expected result).

Reply:
OK, I see, fixed. Thx!